package.json

```diff
@@ -121,5 +124,10 @@
     "webpack-bundle-analyzer": "^4.10.2",
     "webpack-cli": "^6.0.1",
     "webpack-dev-server": "^5.2.0"
+  },
+  "resolutions": {
+    "@types/react": "^18.3.8",
+    "@types/react-dom": "^18.3.0",
+    "monaco-editor": "^0.52.2"
```

Comment:
why we need these resoultions?

Reply:
Subsequent `yarn install` will not show these, but whenever a new package is installed or removed they are flagged again.